From 6f4e4f6eabe4f246d5edace8dc84f6d2dcabd376 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sun, 19 Aug 2018 08:53:49 -0600
Subject: [PATCH 01/10] DO NOT MERGE: klte-common: Requisite bring-up BS change

Currently:
* Permissive
* Kill bcm2079 NFC
* Kill pn547 NFC
* Kill s3fwrn5 NFC
* Don't build SamsungDoze
* Don't build FlipFlap

Change-Id: Iec4e6927e019c9ec91cc16d3eeacb0990ffff6f5
---
 BoardConfigCommon.mk    | 1 +
 klte.mk                 | 4 ++--
 nfc/bcm2079x/board.mk   | 2 +-
 nfc/bcm2079x/product.mk | 6 +++---
 nfc/pn547/board.mk      | 2 +-
 nfc/pn547/product.mk    | 6 +++---
 nfc/s3fwrn5/board.mk    | 2 +-
 nfc/s3fwrn5/product.mk  | 6 +++---
 8 files changed, 15 insertions(+), 14 deletions(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 68355a4..ad54467 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -45,6 +45,7 @@ DEVICE_MANIFEST_FILE += $(COMMON_PATH)/manifest.xml
 # Kernel
 BOARD_KERNEL_BASE := 0x00000000
 BOARD_KERNEL_CMDLINE := console=null androidboot.hardware=qcom user_debug=31 msm_rtb.filter=0x37 ehci-hcd.park=3 zcache.enabled=1 zcache.compressor=lz4 maxcpus=1
+BOARD_KERNEL_CMDLINE += androidboot.selinux=permissive
 BOARD_KERNEL_IMAGE_NAME := zImage
 BOARD_KERNEL_PAGESIZE := 2048
 BOARD_KERNEL_SEPARATED_DT := true
diff --git a/klte.mk b/klte.mk
index 4826d74..a9bcbff 100644
--- a/klte.mk
+++ b/klte.mk
@@ -46,11 +46,11 @@ PRODUCT_AAPT_CONFIG := normal
 PRODUCT_AAPT_PREF_CONFIG := xxhdpi
 
 # Doze
-PRODUCT_PACKAGES += \
+#PRODUCT_PACKAGES += \
     SamsungDoze
 
 # FlipFlap
-PRODUCT_PACKAGES += \
+#PRODUCT_PACKAGES += \
     FlipFlap
 
 # IPv6 tethering
diff --git a/nfc/bcm2079x/board.mk b/nfc/bcm2079x/board.mk
index 4c8a742..e88095e 100644
--- a/nfc/bcm2079x/board.mk
+++ b/nfc/bcm2079x/board.mk
@@ -15,4 +15,4 @@
 #
 
 # NFC
-include $(COMMON_PATH)/nfc/board.mk
+#include $(COMMON_PATH)/nfc/board.mk
diff --git a/nfc/bcm2079x/product.mk b/nfc/bcm2079x/product.mk
index 18ca4e8..f51bb0f 100644
--- a/nfc/bcm2079x/product.mk
+++ b/nfc/bcm2079x/product.mk
@@ -15,14 +15,14 @@
 #
 
 # NFC
-$(call inherit-product, device/samsung/klte-common/nfc/product.mk)
+#$(call inherit-product, device/samsung/klte-common/nfc/product.mk)
 
-PRODUCT_PACKAGES += \
+#PRODUCT_PACKAGES += \
     android.hardware.nfc@1.0-impl-bcm \
     android.hardware.nfc@1.0-service \
     nfc_nci.bcm2079x.default
 
-PRODUCT_COPY_FILES += \
+#PRODUCT_COPY_FILES += \
     device/samsung/klte-common/nfc/bcm2079x/libnfc-brcm.conf:$(TARGET_COPY_OUT_VENDOR)/etc/libnfc-brcm.conf \
     device/samsung/klte-common/nfc/bcm2079x/libnfc-brcm-20791b04.conf:$(TARGET_COPY_OUT_VENDOR)/etc/libnfc-brcm-20791b04.conf \
     device/samsung/klte-common/nfc/bcm2079x/libnfc-brcm-20791b05.conf:$(TARGET_COPY_OUT_VENDOR)/etc/libnfc-brcm-20791b05.conf
diff --git a/nfc/pn547/board.mk b/nfc/pn547/board.mk
index e52b1ab..cc0b517 100644
--- a/nfc/pn547/board.mk
+++ b/nfc/pn547/board.mk
@@ -15,7 +15,7 @@
 #
 
 # NFC
-include $(COMMON_PATH)/nfc/board.mk
+#include $(COMMON_PATH)/nfc/board.mk
 
 # PN547 specific flags
 BOARD_NFC_CHIPSET := pn547
diff --git a/nfc/pn547/product.mk b/nfc/pn547/product.mk
index d34ea24..12f1490 100644
--- a/nfc/pn547/product.mk
+++ b/nfc/pn547/product.mk
@@ -15,13 +15,13 @@
 #
 
 # NFC
-$(call inherit-product, device/samsung/klte-common/nfc/product.mk)
+#$(call inherit-product, device/samsung/klte-common/nfc/product.mk)
 
-PRODUCT_PACKAGES += \
+#PRODUCT_PACKAGES += \
     android.hardware.nfc@1.0-impl \
     libpn547_fw \
     nfc_nci.msm8974
 
-PRODUCT_COPY_FILES += \
+#PRODUCT_COPY_FILES += \
     device/samsung/klte-common/nfc/pn547/libnfc-brcm.conf:$(TARGET_COPY_OUT_VENDOR)/etc/libnfc-brcm.conf \
     device/samsung/klte-common/nfc/pn547/libnfc-nxp.conf:$(TARGET_COPY_OUT_VENDOR)/etc/libnfc-nxp.conf
diff --git a/nfc/s3fwrn5/board.mk b/nfc/s3fwrn5/board.mk
index 4c8a742..e88095e 100644
--- a/nfc/s3fwrn5/board.mk
+++ b/nfc/s3fwrn5/board.mk
@@ -15,4 +15,4 @@
 #
 
 # NFC
-include $(COMMON_PATH)/nfc/board.mk
+#include $(COMMON_PATH)/nfc/board.mk
diff --git a/nfc/s3fwrn5/product.mk b/nfc/s3fwrn5/product.mk
index d4f9497..7bbf86b 100644
--- a/nfc/s3fwrn5/product.mk
+++ b/nfc/s3fwrn5/product.mk
@@ -15,11 +15,11 @@
 #
 
 # NFC
-$(call inherit-product, device/samsung/klte-common/nfc/product.mk)
+#$(call inherit-product, device/samsung/klte-common/nfc/product.mk)
 
-PRODUCT_PACKAGES += \
+#PRODUCT_PACKAGES += \
     android.hardware.nfc@1.0-impl
 
-PRODUCT_COPY_FILES += \
+#PRODUCT_COPY_FILES += \
     device/samsung/klte-common/nfc/s3fwrn5/libnfc-sec.conf:$(TARGET_COPY_OUT_VENDOR)/etc/libnfc-brcm.conf \
     device/samsung/klte-common/nfc/s3fwrn5/libnfc-sec-hal.conf:$(TARGET_COPY_OUT_VENDOR)/etc/sec-nfc.conf
-- 
2.17.1

